src/messages/IPostMessageSentHandler.ts

```diff
@@ -1,26 +1,23 @@
+import { IHttp } from '../accessors/IHttp';
 import { IRead } from '../accessors/index';
-import { IRoom } from '../rooms/index';
-import { IUser } from '../users/index';
 import { IMessage } from './IMessage';
 
 export interface IPostMessageSentHandler {
     /**
      * First step when a handler is executed: Enables the handler to signal
-     * to the Rocketlets framework whether the it shall actually execute for the message
+     * to the Rocketlets framework whether handler shall actually execute for the message
      * about to be sent.
      *
      * @param message The message which is being sent
      * @param read An accessor to the environment
      * @return true: run the pre-logic
      */
-    isMessageApplicable(message: IMessage, read: IRead): boolean;
+    checkPostMessageSent(message: IMessage, read: IRead): boolean;
 
     /**
      * Method called *after* the message is sent to the other clients.
      *
-     * @param user The user who sent the message
-     * @param room The room where the message was sent to
      * @param message The message which was sent
      */
-    postMessageSent(user: IUser, room: IRoom, message: IMessage): void;
+    communicatePostMessageSent(message: IMessage, read: IRead, http: IHttp): void;
```

Comment:
Yes, agreed that the object and then accessors should be provided. My question is with the `http: IHttp` one here, do you think that it should be provided in more of a global scope since several methods could potentially use it.

Let's say someone creates a Rocketlet that uses the `preMessageSent` and wants to check if the sender has paid for sending messages to that channel then if they haven't the sending of the message is cancelled. They would call their own external http service for the checking..